dashboard/client/src/pages/metrics/Metrics.tsx

```diff
@@ -198,6 +198,11 @@ export const Metrics = () => {
               ))}
             </TextField>
           </Paper>
+          <Alert severity="info">
```

Comment:
nit: would be nice if the alert was closable

Reply:
+1. Is it easy to do it?

Reply:
It's not easy to permanently dismiss it. I'll need to store the dismissed state in local storage or in memory of the dashboard API.

I'd rather not work on this right now. I don't think it's too jarring